scrapy/extensions/feedexport.py

```diff
@@ -7,6 +7,7 @@
 import logging
 import os
 import re
+from scrapy.extensions.batches import Batch
```

Comment:
This should be along the rest of imports from `scrapy` below, and keeping the alphabetical order.

Reply:
I have no idea how it got there really :laughing: